src/sqlancer/questdb/QuestDBSchema.java

```diff
@@ -142,6 +147,8 @@ public String toString() {
                 }
             case BOOLEAN:
                 return Randomly.fromOptions("BOOLEAN");
+            case SYMBOL:
+                return Randomly.fromOptions("SYMBOL");
```

Comment:
suggestion for a refactoring PR: if we only have one option, we can directly return the string (e.g., `return "SYMBOL"`).